ansible/fhir-stress-test-temp-testers.yml

```diff
@@ -0,0 +1,110 @@
+---
+##
+# Provisions temporary test clients and servers in the active AWS account
+# Runs the stress test, and tears down test clients and servers
+
+- name: AWS - Provision Resources and Instances
+  hosts: localhost
+  connection: local
+  gather_facts: false
+
+  vars:
+    # "Compute Optimized" with 4 vCPUs, 7.5 GB RAM, $0.209/hour
+    ec2_instance_type: "c4.xlarge"
+  
+  tasks:
+    
+    - name: Local - whoami
+      local_action: command whoami
+      register: whoami
+
+    #- name: Acquire a session token
```

Comment:
Could you add a `TODO` so that we have a marker pointing us to fix this in the future?